### DISCUSSION TITLE
Break timer

### DISCUSSION THREAD

**Discussion by Qasym** [super-productivity/super-productivity#1946] [General]
Dear, super-productivity creators!
Thank you for developing such a wonderful app!

I have one issue with the break timer in the 7.10.1 version of the app. The issue is the break time notifications.
Let's say I am currently focusing on a task, then I decide to take a break, I pause the work timer, however app still thinks that I am working and notifies me to take a break. 
That sometimes become really annoying, is there a way to keep receiving break notifications but not when the work timer is off?

**Comment by johannesjo** [comment]
Hey there! You mean with the pomodoro integration, right? I think there is already an open issue somewhere, though I am unable to find it now.... :)

**Reply by Qasym** [reply]
No, it's not the pomodoro integration.

When you add tasks for today, you can start focus session and it tracks your time. With default settings, after an hour, you receive a notification (something of a similar meaning) "you've been focusing without break for 1h, take a break to be efficient in the long run".

My issue is, I do follow the advice in the notification and pause my focus session, however the app doesn't count it as a break.

**Reply by johannesjo** [reply]
Ah I see! Thanks for the explanation! There are two things that reset the break reminder (maybe the translation is also a bit off, not sure):
1. The idle dialog appears and you choose break there
2. You press on the little clock icon on the right of "Without break: x"

**Comment by bonnebulle** [comment]
Hey, am new user ( an I take a lot of breaks )
I create a **Chronometre Counter** for my breaks.
When I click on, I would like to active the Break Timer... 
I find options to start the Pomodoro pause timer... or a specific task but not the SP breaker timer.

-> Say to SP am doing a pause (am unfocused right now) should help SP not ask me to take a break when am allready a full breaker...

An other option should be : 
Add an option in the popup for not tracked time to say, its was a break.

Thanks you are amazing
V

**Comment by Lobodiurno** [comment]
**Related question**
Let's say I've finished working on a feature. I grab a coffee and stop the timer. When I return to my computer, this screen appears. How can we classify this time as a break?

How do we tell the app that we were on a break?.
 
 I guess  this pop up appears because I have activated the option "only trigger IDLE time dialog when the current task is selected". Anyway I would suggest to have a button to also allow this time to go to break.

<img width="1249" height="391" alt="ksnip_20260324-095703" src="https://github.com/user-attachments/assets/286b0e46-68b2-40dc-beea-c4b2b6d1504f" />

The following question: How do I take a break? #1286 is also related.

**Comment by mcassaniti** [comment]
I want to track and display (or report on):

* Time spent working on tasks which is already covered
* Time spent taking a break:
  * Total time
  * Detailed view of how long each break was
* A way to add breaks since I may be browsing a website as a break
* Missing time during my day which likely means I was distracted doing something else

It seems this would require a new plugin at this point, but any suggestions are welcome.

**Comment by Asmariahendrick** [comment]
I found this thread because I was just looking for a 'break' button. I would love to be able to just click when I am on a break. I use the timers when i work, but not all of my tasks are timed. (like take your meds, I don't time that, lol, I just check it off). So I'll have this weird flow when I just want to click 'take a break' where needed. Not sure if this applies to the discussion but since it's about breaks, I wasn't sure where else to put it.
